src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,32 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import { UserInfo } from '../UserInfo/UserInfo';
+import { CommentList } from '../CommentList';
+import './PostInfo.scss';
+
+export const PostInfo = ({ todo }) => (
```

Comment:
Destructure `todo`

Reply:
not fixed
<img width="502" alt="image" src="https://github.com/mate-academy/react_static-list-of-posts-js/assets/98764641/3a412f77-456e-4ae3-badd-f264d7b7205d">